### PR TITLE
Open sgid blogpost

### PR DESCRIPTION
**Pull request by steveoh**
temp spot for blog post.

### CODE REVIEW

**Comment by steveoh** [comment]
You're going to need to resize that image. I don't think it adds much value so i might even remove it all together.

**Comment by steveoh** [comment]
Please use the markdown conventions from https://github.com/agrc/gis.utah.gov/blob/open-sgid-blogpost/README.md#markdown-syntax-conventions

**Comment by steveoh** [comment]
you should change the file name also if we are publishing today

File: _posts/2020-05-04-introducing-open-sgid.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+A nearly month-long beta testing period was helpful to identify several issues and some inherent differences between the Open SGID and the enterprise geodatabase of the SGID10 era. A [GitHub repository](https://github.com/agrc/open-sgid) was created as the primary source of information for the Open SGID, and to catalog and address [these issues and differences](https://github.com/agrc/open-sgid/issues) with explanations or troubleshooting techniques.
+
+Known issues:
+- [Tables do not contain metadata](https://github.com/agrc/open-sgid/issues/5)
+- [Fields with domains currently only display coded values](https://github.com/agrc/open-sgid/issues/7)
+- [Some geoprocessing tools may fail for not having an OID field (client-specific)](https://github.com/agrc/open-sgid/issues/14)
```
Comment (steveoh) [inline _posts/2020-05-04-introducing-open-sgid.md:32]: i might remove this as it is a non issue in the latest pro version.

Comment (nathankota) [reply _posts/2020-05-04-introducing-open-sgid.md:32]: please alter however you see fit

**Pull request merged**